src/iop/atrous.c

```diff
@@ -1529,19 +1529,24 @@ static gboolean area_scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer
 
   if(dt_gui_ignore_scroll(event)) return FALSE;
 
-  int delta_y;
-  if(dt_gui_get_scroll_unit_deltas(event, NULL, &delta_y))
+  if(dt_modifier_is(event->state, GDK_CONTROL_MASK))
   {
-    if(dt_modifier_is(event->state, GDK_CONTROL_MASK))
+    int delta_y;
+    if(dt_gui_get_scroll_unit_deltas(event, NULL, &delta_y))
     {
       //adjust aspect
       const int aspect = dt_conf_get_int("plugins/darkroom/atrous/aspect_percent");
       dt_conf_set_int("plugins/darkroom/atrous/aspect_percent", aspect + delta_y);
     }
-    else
+  }
+  else
+  {
+    gdouble delta_y;
+    if(dt_gui_get_scroll_deltas(event, NULL, &delta_y))
+    {
       c->mouse_radius = CLAMP(c->mouse_radius * (1.0 + 0.1 * delta_y), 0.25 / BANDS, 1.0);
-
-    gtk_widget_queue_draw(widget);
+      gtk_widget_queue_draw(widget);
+    }
```

Comment:
Is there a reason that you've changed this? It previously redrew the widget regardless of whether the Ctrl modifier was selected. Does it still work correctly after you've made this change? Similarly in other functions below.

Reply:
The change wasn't intentional. Everything seems to work correctly but I'll change this back to the way it was before.